.github/workflows/integration-tests.yml

```diff
@@ -49,6 +49,10 @@ jobs:
         id: scylla
         shell: bash
         run: |
+          ulimit -n 65536
+          sudo sysctl -w fs.aio-max-nr=30000000
```

Comment:
What about https://opensource.docs.scylladb.com/stable/kb/scylla-limits-systemd.html#before-you-start ?

Reply:
It is done for holding fs to lot's of sstables, our tests are not suppose to generate lot's of them.